Packages/vcs/Lib/VTKPlots.py

```diff
@@ -357,7 +358,7 @@ def createRenderer(self, *args, **kargs):
         return ren
 
     def update(self, *args, **kargs):
-        self._lastSize = -1
+        self._lastSize = None
```

Comment:
@doutriaux1 There are a bunch of test failures because of small differences in the baselines - that happens on master branch as well. I will push _xx versions shortly. Everything else looks good.